src/main/java/com/dhex/shipping/model/ActivityIndicatorEnum.java

```diff
@@ -0,0 +1,36 @@
+package com.dhex.shipping.model;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Created by Jorge Santa Cruz on 23/10/2016.
+ */
+public enum ActivityIndicatorEnum {
+    ENABLE(1L), DISABLE(0L), ALL(-1L);
+
+    private Long id;
+    private ActivityIndicatorEnum(long id){
+        this.id = id;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public static Long getIdByBooleanStatus(boolean status){
+        return status ? 1L : 0L;
+    }
+
+    public static ActivityIndicatorEnum getActivityIndicatorEnumById(Long id) {
```

Comment:
we can name this method by removing the redundant part of the name. it would finish with something like this:
```java
public static ActivityIndicatorEnum obtainById(Long id) {
```